### PR TITLE
Add a note to release-4.0.9.md

### PR DESCRIPTION
**Pull request by HydraLee**
更新release note 中的dumpling的压缩备份说明

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-4.0.9.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ TiDB 版本：4.0.9
         - 支持同时设置 `-F` 和 `-r` 两个参数 [#177](https://github.com/pingcap/dumpling/pull/177)
         - 默认不导出系统表 [#194](https://github.com/pingcap/dumpling/pull/194)
         - 在设置 `--transactional-consistency` 参数时支持重建 MySQL 链接 [#199](https://github.com/pingcap/dumpling/pull/199)
+        - 支持使用 -c,--compress 参数用于指定dumpling是否使用压缩备份及指定压缩算法 [#202](https://github.com/pingcap/dumpling/pull/202)
```
Comment (lichunzhu) [inline releases/release-4.0.9.md:107]:
```suggestion
        - 支持使用 -c,--compress 参数指定 dumpling 是否使用的压缩算法，空字符串代表不压缩 [#202](https://github.com/pingcap/dumpling/pull/202)
```

**Review by lichunzhu — COMMENTED** [review]
Rest LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**